src/genie/libs/parser/iosxe/show_interface.py

```diff
@@ -297,11 +297,12 @@ def cli(self,interface="",output=None):
         # auto-duplex, 10 Gb/s, media type is 10G
         # Full Duplex, 10000Mbps, link type is force-up, media type is SFP-LR
         # Full-duplex, 100Gb/s, link type is force-up, media type is QSFP 100G SR4
+        # Full-duplex, 10Gb/s, media type is 100/1000/2.5G/5G/10GBaseTX
         p11 = re.compile(r'^(?P<duplex_mode>\w+)[\-\s]+[d|D]uplex\, '
                          r'+(?P<port_speed>[\w\s\/]+|[a|A]uto-[S|s]peed|Auto '
                          r'(S|s)peed)(?:(?:\, +link +type +is '
                          r'+(?P<link_type>\S+))?(?:\, *(media +type +is| )'
-                         r'*(?P<media_type>[\w\/\- ]+)?)(?: +media +type)?)?$')
+                         r'*(?P<media_type>[\w\/\-. ]+)?)(?: +media +type)?)?$')
```

Comment:
. is used when any characters are considered valid. Since you want it to match to a period, you need to escape it.
It would look like this:

 ```suggestion
                         r'*(?P<media_type>[\w\/\-\. ]+)?)(?: +media +type)?)?$')
```

Reply:
@domachad  Thank you for the help.  I've made the updates as suggested.  I've run the folder_parsing locally and all tests passed.
Feels good to be able to contribute.